generators/dbpedia_openai_1M/generate_dbpedia_openai_1M.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os
+import random
+from typing import Iterable, Tuple
+
+import pandas as pd
+import numpy as np
+import tqdm
+
+from pathlib import Path
+from datasets import load_dataset
+from generators.config import DATA_DIR
+from generators.search_generator.qdrant_generator import index_qdrant, search_qdrant
+
+SAMPLE_SIZE = 100_000
```

Comment:
@generall this was wrong. It should have been 1M.

Reply:
Resolved this. It will now store `975k` vectors in the DB during the benchmark and query/test with `5k` embeddings.